helpers/voting/getVoteMetaData.ts

```diff
@@ -44,6 +44,30 @@ export function getVoteMetaData(
     };
   }
 
+  const isAcross = decodedIdentifier === "ACROSS-V2";
+  if (isAcross) {
```

Comment:
This is fine for now but it might make sense to have some external module where we put these kinds of encodings/ augmentations so that it does not live within the `getVoteMetaData file.